packages/ui/primitives/document-flow/add-signers.tsx

```diff
@@ -174,6 +164,20 @@ export const AddSignersFormPartial = ({
     removeSigner(index);
   };
 
+  const onAddSelfSigner = () => {
```

Comment:
Did we specifically move this function for a reason? Makes the diff unnecessarily larger

Also add some spacing between these lines in the function for readability

Reply:
Yes, it was on purpose as I wanted to onRemoveSigner() after its declaration